### PR TITLE
Approve CWL v1.2 stable release

### PR DESCRIPTION
**Pull request by tetron**
Submitted approval, CWL 1.2 stable release.

### CODE REVIEW

File: contrib.md [hunk 1]
```diff
@@ -13,17 +13,12 @@ Contributors to v1.2:
 
 Incorporates the work of past authors and contributors to [CWL v1.0](https://www.commonwl.org/v1.0/Workflow.html) and [CWL v1.1](https://www.commonwl.org/v1.1/Workflow.html).
 
-This specification is a draft, and has not yet been approved as an official CWL standard.
-
-<!-- v1.2.0-devN is not yet approvied
-This specification was [approved on 6 June 2019](https://github.com/common-workflow-language/common-workflow-language/issues/861) by the CWL leadership team consisting of:
+This specification was [approved on XXX August 2020](https://github.com/common-workflow-language/cwl-v1.2/XXX) by the CWL leadership team consisting of:
```
Comment (mr-c) [inline contrib.md:16]: TODO, after approval

**Comment by tetron** [comment]
Brandi voted for approval by email.

**Comment by mr-c** [comment]
We now have a majority!

**Comment by jmchilton** [comment]
Belated +1 from me - really nice work y'all - you've got my constant gratitude. Very impressive and very exciting.

**Pull request merged**